PR_REVIEW.md

```diff
@@ -18,6 +18,7 @@ For modules and core platform updates, the initial reviewer should request an ad
 - Once there is 2 `LGTM` on the PR, merge to master
 - Ask the submitter to add a PR for documentation if applicable.
 - Add a line into the [draft release](https://github.com/prebid/Prebid.js/releases) notes for this submission. If no draft release is available, create one using [this template]( https://gist.github.com/mkendall07/c3af6f4691bed8a46738b3675cb5a479)
+- Add the PR to the appropriate project board (I.E. 1.23.0 Release) for the week.
```

Comment:
for consistency, and to make it easy to find, might want to add a link to https://github.com/prebid/Prebid.js/projects